site/web-stomp.md

```diff
@@ -154,26 +156,27 @@ Erlang/OTP 17.5 or a later version.
 TLS (SSL) configuration parameters are provided in the `ssl_config` section:
 
 <pre class="sourcecode ini">
-web_stomp.ssl.port       = 12345
```

Comment:
`web_stomp.ssl.*` are [also supported](https://github.com/rabbitmq/rabbitmq-web-stomp/blob/v3.7.x/priv/schema/rabbitmq_web_stomp.schema). This should be discussed with specific examples on the mailing list first. Thank you.